internal/handlers/common/update.go

```diff
@@ -526,6 +673,10 @@ func ValidateUpdateOperators(update *types.Document) error {
 		return err
 	}
 
+	if err = checkConflictingOperators(mul, currentDate, inc, min, max, pop, set, setOnInsert, unset); err != nil {
```

Comment:
We could not do this. Because this is gonna get outdated very quickly.

While testing, I discovered each update operator conflict to each other and this is checking that,